snmp/README.md

```diff
@@ -133,300 +133,24 @@ If necessary, additional metrics can be defined in the instances. These metrics
-| `snmp.ifHCOutBroadcastPkts`        | The total number of packets that higher-level protocols requested be transmitted addressed to a broadcast address at this sub-layer including those discarded or not sent.                                                                                                | `interface`               |
-| `snmp.tcpActiveOpens`              | The number of times that TCP connections have made a direct transition to the SYN-SENT state from the CLOSED state.                                                                                                                                                       |                           |
-| `snmp.tcpPassiveOpens`             | The number of times TCP connections have made a direct transition to the SYN-RCVD state from the LISTEN state.                                                                                                                                                            |                           |
-| `snmp.tcpAttemptFails`             | The number of times that TCP connections have made a direct transition to the CLOSED state from either the SYN-SENT state or the SYN-RCVD state, plus the number of times that TCP connections have made a direct transition to the LISTEN state from the SYN-RCVD state. |                           |
-| `snmp.tcpEstabResets`              | The number of times that TCP connections have made a direct transition to the CLOSED state from either the ESTABLISHED state or the CLOSE-WAIT state.                                                                                                                     |                           |
-| `snmp.tcpCurrEstab`                | The number of TCP connections for which the current state is either ESTABLISHED or CLOSE-WAIT.                                                                                                                                                                            |                           |
-| `snmp.tcpHCInSegs`                 | The total number of segments received, including those received in error.                                                                                                                                                                                                 |                           |
-| `snmp.tcpHCOutSegs`                | The total number of segments sent, including those on current connections but excluding those containing only retransmitted octets.                                                                                                                                       |                           |
-| `snmp.tcpRetransSegs`              | The total number of segments retransmitted; that is, the number of TCP segments transmitted contai
```

Comment:
Change to: `Metrics collected depends on the integration being configured with the corresponding profile`

Reply:
```suggestion
The SNMP check submits specified metrics under the `snmp.*` namespace. **Metrics collected depends on the integration being configured with the corresponding profile**.
```